dev/com.ibm.ws.cdi.visibility_fat/fat/src/com/ibm/ws/cdi/visibility/tests/vistest/VisTest.java

```diff
@@ -731,10 +731,7 @@ public static void afterClass() throws Exception {
         if (server != null) {
             server.stopServer();
         }
-    }
 
-    @AfterClass
-    public static void cleanupFeatures() throws Exception {
```

Comment:
Revert this change. Doing it as separate methods ensures the uninstall of features is still done even if the `stopServer()` throws an exception.